routes/actions.js

```diff
@@ -341,25 +344,52 @@ router.post('/register/:code', function(req, res, next) {
 
 });
 
-function sendRegistrationForm(req, res, formInfo) {
+async function sendRegistrationForm(req, res, formInfo) {
```

Comment:
Marking a function `async` means that it will return a promise -- and all invocations of the function have to be changed to account for that. This is probably not what you want to do here.

Reply:
My understanding is that I need the function to be async to allow usage of await within it.  It's in one the promise tutorials you sent: [ponyfoo async await](https://ponyfoo.com/articles/understanding-javascript-async-await#using-async-await).

Reply:
That's true, but it changes the return value to a promise, so it has consequences for all callers of the function. You can update all callers to account for this fact, refactor the function into smaller ones, or avoid the use of `await`.

Reply:
Poking a bit more at this particular case, it looks like the `async` change is probably safe here, since the callers don't depend on the return value of the function, which is always `undefined`. If it passes tests and the PR is complete on your end, I'll poke a bit more to see if there are any side effects of this change.